From c64e58cd7718d93fe5f63cc4ecad32b9eea00bff Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 24 Oct 2023 10:35:28 +0800
Subject: [PATCH 2/3] dts: cvc-fl: disable can0 device node

The flexcan0 device is used by m7, so disable it from A53 side.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi b/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi
index bc60344f93ed1..5a12262900528 100644
--- a/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi
@@ -195,7 +195,7 @@ &usdhc0 {
 &can0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&can0_pins>;
-	status = "okay";
+	status = "disabled";
 };
 
 &can1 {
-- 
2.36.1

